frontend/src/components/ShootVersion/ShootVersionUpdate.vue

```diff
@@ -93,14 +101,21 @@ export default {
             return -1
           } else if (b.header) {
             return 1
-          } else {
-            if (semver.eq(a.version, b.version)) {
-              return 0
-            } else if (semver.gt(a.version, b.version)) {
-              return 1
-            } else {
+          }
+          if (semver.diff(a.version, b.version) === 'patch') {
```

Comment:
The versions are already grouped by update type `['patch', 'minor', 'major']`. I would propose to sort the versions for each group and afterwards add the header to the group. In this case all the entries would not need to have the type property I think.

Reply:
How do we handle `pre` release versions?